### DISCUSSION TITLE
UPDATE August 2022

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#234] [General]
Just use this mappings. If you don't know about mappings then see the home page and learn how to use it first.

NOTE : 
Use Snapchat **11.91.1.39**
[84634.zip](https://github.com/rodit/SnapMod/files/9332956/84634.zip)

Use Snapchat **11.92.0.33** (Latest as on 18th of August 2022)
[84635.zip](https://github.com/rodit/SnapMod/files/9372987/84635.zip)
- Fixed : Showing message content in Notifications
- Fixed : Auto Save Messages
- Fixed : Many more features

[chat messages saving fixed.zip](https://github.com/rodit/SnapMod/files/9381102/chat.messages.saving.fixed.zip)
New Apk for Snapchat 11.91.1.39 and above to fix auto save messages features which was not working. It took me almost 2 days to figure out the working and I edited the java bytecode(smali) to make it work as the android studio project can't be compiled which is given as source code. So, anyways, I fixed chat saving. 
Will be providing more updates since @rodit comes back.

**Comment by RevealedSoulEven** [comment]


**Reply by RickyXO2** [reply]
I'm sure @rodit is well aware of these bugs. He's busy with his personal life right now & will or will not update whenever he very well pleases & it is quite rude to demand an update, especially for an app that is free for you to use. You're the 2nd person I've seen demand an update & it's people like you why some devs put an end to their projects.

**Reply by RevealedSoulEven** [reply]
Please don't take it the wrong way, I'm not so eager for it. I'm just notifying about the bug. Sorry if that hurts. I also know it's a personal project so I'm also hoping for new updates

**Comment by authorisation** [comment]
Can you share your snap.ds? I've tried updating the schematic myself with no success.

**Comment by RevealedSoulEven** [comment]
I'll send it later, it's in another device. Btw I did some modifications to
match the classes but still I fixed only few of them not all

**Reply by authorisation** [reply]
Thank you :)

**Reply by authorisation** [reply]
Can you look into it now?

**Reply by RevealedSoulEven** [reply]
[snap_fix.zip](https://github.com/rodit/SnapMod/files/9351299/snap_fix.zip)

Here it is. Sorry for being late, but ya it's not completely fixed yet, I'm new to these and I don't understand the obfuscation of the snapchat apk although reading the [Wiki  of dexsearch](https://github.com/rodit/dexsearch/wiki/Writing-Schematics) I understood something on my own but alas I could only fix 3 or 4 functions out of 10 functions which gave error. So these 6 functions are yet to be fixed!
![image](https://user-images.githubusercontent.com/49313374/184889469-cbe18266-d262-40ef-95fe-7599e059fb52.png)

**Reply by authorisation** [reply]
Thanks :)

**Reply by RevealedSoulEven** [reply]
Try fixing more 🥺🤧

**Reply by RevealedSoulEven** [reply]
You can msg me on telegram `@soulevenofficial`

**Reply by Sadyantssr** [reply]
Sir, can you please share the latest version mappings (84635)?

**Comment by RevealedSoulEven** [comment]
Sure. Just give me an hour

**Comment by RSR1337** [comment]
> Just use this mappings. If you don't know about mappings then see the home page and learn how to use it first.
> 
> NOTE : Use Snapchat **11.91.1.39** [84634.zip](https://github.com/rodit/SnapMod/files/9332956/84634.zip)
> 
> Use Snapchat **11.92.0.33** (Latest as on 18th of August 2022) [84635.zip](https://github.com/rodit/SnapMod/files/9372987/84635.zip)
> 
> * Fixed : Showing message content in Notifications
> * Fixed : Auto Save Messages
> * Fixed : Many more features
> 
> [chat messages saving fixed.zip](https://github.com/rodit/SnapMod/files/9381102/chat.messages.saving.fixed.zip) New Apk for Snapchat 11.91.1.39 and above to fix auto save messages features which was not working. It took me almost 2 days to figure out the working and I edited the java bytecode(smali) to make it work as the android studio project can't be compiled which is given as source code. So, anyways, I fixed chat saving. Will be providing more updates since @rodit comes back.

Ur mapping work best with SnapMod 1.8.2

**Reply by RevealedSoulEven** [reply]
Use that with the snapmod apk I uploaded (save chat fix)

**Reply by RSR1337** [reply]
I tried 1.8.3 it's issued, If you can modify version 1.8.2 so it will be better

**Reply by RSR1337** [reply]
Also if you can fix chat context menu, I would be thankful to you ✨

**Comment by RevealedSoulEven** [comment]
I'm working on it. Can you please elaborate what's working fine in 1.8.2
instead of 1.8.3?

**Reply by RSR1337** [reply]
> I'm working on it. Can you please elaborate what's working fine in 1.8.2 instead of 1.8.3?
> […](#)
> On Fri, 19 Aug, 2022, 9:43 pm iRashiD-AE, ***@***.***> wrote: Also if you can fix chat context menu, I would be thankful to you ✨ — Reply to this email directly, view it on GitHub <[#234 (reply in thread)](https://github.com/rodit/SnapMod/discussions/234#discussioncomment-3432904)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALYHMXUCIAUGMOUPQAK2PKTVZ6XELANCNFSM56PPEPPA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

There is no issue with the disappearance of stories and subscribers and discover on 1.8.2 , Everyone has this issue on 1.8.3

**Comment by RevealedSoulEven** [comment]
I'll give it a try. I can fix it

**Comment by jkooper32** [comment]
> Just use this mappings. If you don't know about mappings then see the home page and learn how to use it first.
> 
> NOTE : Use Snapchat **11.91.1.39** [84634.zip](https://github.com/rodit/SnapMod/files/9332956/84634.zip)
> 
> Use Snapchat **11.92.0.33** (Latest as on 18th of August 2022) [84635.zip](https://github.com/rodit/SnapMod/files/9372987/84635.zip)
> 
> * Fixed : Showing message content in Notifications
> * Fixed : Auto Save Messages
> * Fixed : Many more features
> 


Great work !!! Keep it up and thank you !!

**Comment by melbarouzi** [comment]
> Just use this mappings. If you don't know about mappings then see the home page and learn how to use it first.
> 
> NOTE : Use Snapchat **11.91.1.39** [84634.zip](https://github.com/rodit/SnapMod/files/9332956/84634.zip)
> 
> Use Snapchat **11.92.0.33** (Latest as on 18th of August 2022) [84635.zip](https://github.com/rodit/SnapMod/files/9372987/84635.zip)
> 
> * Fixed : Showing message content in Notifications
> * Fixed : Auto Save Messages
> * Fixed : Many more features
> 
> [chat messages saving fixed.zip](https://github.com/rodit/SnapMod/files/9381102/chat.messages.saving.fixed.zip) New Apk for Snapchat 11.91.1.39 and above to fix auto save messages features which was not working. It took me almost 2 days to figure out the working and I edited the java bytecode(smali) to make it work as the android studio project can't be compiled which is given as source code. So, anyways, I fixed chat saving. Will be providing more updates since @rodit comes back.

Great Work   .... can u tell me the features  are working and not

I wil appreciate it .

**Comment by RevealedSoulEven** [comment]
See my github repositories there is SnapMod-new and in it everything is
working fine except the new context chat menu.

**Reply by RSR1337** [reply]
Here is for stable SC version

[84637.zip](https://github.com/rodit/SnapMod/files/9404439/84637.zip)

**Reply by authorisation** [reply]
mind sharing your snap.ds?

**Reply by RSR1337** [reply]
thank you for ur great work, testing...

**Reply by RevealedSoulEven** [reply]
Your welcome. Are you using my snap.ds or your own customised?

**Reply by RSR1337** [reply]
> mind sharing your snap.ds?

I used ur snap.ds , but stories, subs , discover Still disappear

**Reply by authorisation** [reply]
> > mind sharing your snap.ds?
> 
> I used ur snap.ds , but stories, subs , discover Still disappear

Which SnapMod version?

**Reply by RSR1337** [reply]
> > > mind sharing your snap.ds?
> > 
> > 
> > I used ur snap.ds , but stories, subs , discover Still disappear
> 
> Which SnapMod version?

1.8.4

**Reply by authorisation** [reply]
> > > > mind sharing your snap.ds?
> > > 
> > > 
> > > I used ur snap.ds , but stories, subs , discover Still disappear
> > 
> > 
> > Which SnapMod version?
> 
> 1.8.4

Try 1.8.2 I saw people saying that fixes it

**Reply by RSR1337** [reply]
> > > > > mind sharing your snap.ds?
> > > > 
> > > > 
> > > > I used ur snap.ds , but stories, subs , discover Still disappear
> > > 
> > > 
> > > Which SnapMod version?
> > 
> > 
> > 1.8.4
> 
> Try 1.8.2 I saw people saying that fixes it

I already tried, no context menu and SC crashing when u save opened snaps

**Reply by RSR1337** [reply]
> > > mind sharing your snap.ds?
> > 
> > 
> > I used ur snap.ds , but stories, subs , discover Still disappear
> 
> Which SnapMod version?

Did you modify snapmod 1.8.3 or 1.8.2 ?

**Reply by Lucky-069** [reply]
> See my github repositories there is SnapMod-new and in it everything is working fine except the new context chat menu.
> […](#)
> On Tue, 23 Aug, 2022, 6:09 pm melbarouzi, ***@***.***> wrote: Just use this mappings. If you don't know about mappings then see the home page and learn how to use it first. NOTE : Use Snapchat *11.91.1.39* 84634.zip <https://github.com/rodit/SnapMod/files/9332956/84634.zip> Use Snapchat *11.92.0.33* (Latest as on 18th of August 2022) 84635.zip <https://github.com/rodit/SnapMod/files/9372987/84635.zip> - Fixed : Showing message content in Notifications - Fixed : Auto Save Messages - Fixed : Many more features chat messages saving fixed.zip <https://github.com/rodit/SnapMod/files/9381102/chat.messages.saving.fixed.zip> New Apk for Snapchat 11.91.1.39 and above to fix auto save messages features which was not working. It took me almost 2 days to figure out the working and I edited the java bytecode(smali) to make it work as the android studio project can't be compiled which is given as source code. So, anyways, I fixed chat saving. Will be providing more updates since @rodit <https://github.com/rodit> comes back. Great Work .... can u tell me the features are working and not I wil appreciate it . — Reply to this email directly, view it on GitHub <[#234 (comment)](https://github.com/rodit/SnapMod/discussions/234#discussioncomment-3456265)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALYHMXRUGU65WMTUUF7PMU3V2TBAZANCNFSM56PPEPPA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

I saw this too. Why does the new chat ui not work. I did see some commit. Why was it deleted!

**Comment by RSR1337** [comment]
Almost everything is working on this version

Snapchat version:
https://www.apkmirror.com/apk/snap-inc/snapchat/snapchat-11-90-1-31-release/snapchat-11-90-1-31-2-android-apk-download/

SnapMod version:
https://github.com/rodit/SnapMod/releases/download/v1.8.2/app-release.apk

Mapping:
[84632.zip](https://github.com/rodit/SnapMod/files/9430979/84632.zip)

**Reply by melbarouzi** [reply]
> Almost everything is working on this version
> 
> Snapchat version: https://www.apkmirror.com/apk/snap-inc/snapchat/snapchat-11-90-1-31-release/snapchat-11-90-1-31-2-android-apk-download/
> 
> SnapMod version: https://github.com/rodit/SnapMod/releases/download/v1.8.2/app-release.apk
> 
> Mapping: [84632.zip](https://github.com/rodit/SnapMod/files/9430979/84632.zip)

Pin Conversation  Didn't  Work

**Reply by RSR1337** [reply]
Yes , some features is not working

**Comment by rodit** [comment]
Thanks for providing these updates, I hope to provide an official update for SnapMod soon

**Reply by RevealedSoulEven** [reply]
Really Appreciating for me sir

**Comment by RevealedSoulEven** [comment]
For now I've fixed some issues in the previous version in my forked version. If anyone want to use it every feature in working great there. Check my profile.

**Reply by RSR1337** [reply]
Great work 👍 but you forget to add mapping

Testing...

**Comment by RevealedSoulEven** [comment]
Ahh sorry, I'm adding it wait

**Comment by ghost** [comment]
I finished the last version 12.3.0.22, I will post it soon, after testing it to make sure there are no problems

**Reply by RevealedSoulEven** [reply]
Please send me msg on telegram. Link in my bio

**Comment by RevealedSoulEven** [comment]
Did you fixed those video bugs? If those are fixed, then please let me know

**Reply by ghost** [reply]
> Did you fixed those video bugs? If those are fixed, then please let me know
> […](#)
> On Wed, 12 Oct, 2022, 3:44 pm Me11222, ***@***.***> wrote: I finished the last version 12.3.0.22, I will post it soon, after testing it to make sure there are no problems — Reply to this email directly, view it on GitHub <[#234 (comment)](https://github.com/rodit/SnapMod/discussions/234#discussioncomment-3859007)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALYHMXSODA7EXCODSON3BSLWC2FQVANCNFSM56PPEPPA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

I didn't understand what you mean, if you mean to send the video, even if it is solved there is another problem, for example if you want to send a 15 second video, only the first 10 seconds will be sent. This issue exists from version 11.93 to the latest version

**Reply by RevealedSoulEven** [reply]
That's due to we need to change hooks to match new api. Btw you're right, for that reason I'm using snapchat 11.92.0.33

**Reply by ghost** [reply]
But if you send the video via the timeline button, it will be sent in full, regardless of its duration, which means we don't need to send it through memories.

**Reply by RevealedSoulEven** [reply]
No sending video fails. No matters it's sent from camera roll or recorded via snapchat camera

**Comment by RSR1337** [comment]
@RevealedSoulEven  idk why I still getting this...
![Screenshot_20221012-143308_Snapchat](https://user-images.githubusercontent.com/52901789/195320661-50a636b5-4d4e-4e27-a3ff-7d29b55c64a2.png)

**Comment by RevealedSoulEven** [comment]
Yes. Just go to "Hide Story Sections" and select For You and Subscriptions

**Reply by RSR1337** [reply]
For You section still invisible...

**Comment by RevealedSoulEven** [comment]
Because you hidden it🤣🤣🤣

**Reply by RSR1337** [reply]
No I didn't, I tried to check it and uncheck it but it still invisible , can you screenshot ur?

**Comment by RevealedSoulEven** [comment]
From the hide story option
If you select any option, it will be hidden.
So better uncheck For You if you need it.
But one of the three must be checked. So select Subscriptions and leave the
other 2 unchecked

**Reply by RSR1337** [reply]
Still same
